doc/installation_and_administration_guide/install_from_sources.md

```diff
@@ -106,25 +112,41 @@ If the dependencies are included in the built Cygnus package, then nothing has t
 [Top](#top)
 
 ##<a name="installdeps"></a>Installing dependencies
+###<a name="cygnusdeps"></a>Cygnus dependencies
+These are the packages you will need to install under `APACHE_FLUME_HOME/plugins.d/cygnus/libext/` **if you did not included them in the Cygnus package**:
+
+|       Cygnus dependencies        |
+|:---------------------------------------:|
+|       flume-ng-core-1.4.0.jar      |
```

Comment:
A second coulmn regarding the version is required. For instance:

| Cygnus dependencies | Version |
|---|
| flume.ng-core | 1.4.0 |
| etc | |

Please, apply this comment to rest of the document.